command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 02:49:18PM -0700, Mike Brown wrote:
> Ok... I must have missed the memo where we decided to go back on the
> premise of supporting other Ux choices…

We are not [1].  We're just doing this:

> Note: I never got the impression that anyone thought we could test
> compliance at the interface level without interface spec(s).

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72703237